spec/rights_api_spec.rb

```diff
@@ -87,38 +94,40 @@ def valid_json?(json)
   describe "/rights" do
     context "with a valid HTID" do
       before(:each) { get(rights_api_endpoint + "rights/test.pd_google") }
-      it_behaves_like "rights response"
       it_behaves_like "valid response"
+      it_behaves_like "nonempty response"
     end
 
     context "with an invalid HTID" do
       before(:each) { get(rights_api_endpoint + "rights/bogus.no_such_id") }
-      it_behaves_like "rights response"
+      it_behaves_like "valid response"
       it_behaves_like "empty response"
     end
 
     context "with no HTID" do
       before(:each) { get(rights_api_endpoint + "rights") }
-      it_behaves_like "404 response"
+      it_behaves_like "valid response"
+      it_behaves_like "nonempty response"
```

Comment:
I didn't touch much of anything in the tests because of the companion ticket DEV-992 is specifically about testing. I put a note in that ticket to make shared examples for individual tables and look more closely at content in addition to structure. For this ticket I'll say it's out of scope.